mnelab/model.py

```diff
@@ -131,22 +131,39 @@ def find_events(self, stim_channel, consecutive=True, initial_event=True,
     @data_changed
     def events_from_annotations(self):
         """Convert annotations to events."""
-        events, _ = mne.events_from_annotations(self.current["data"])
+        events, mappings = mne.events_from_annotations(self.current["data"])
+        # Mappings for annots are swapped {str: int} should be {int: str}
+        mappings = dict((v, k) for k, v in mappings.items())
         if events.shape[0] > 0:
             self.current["events"] = events
-            self.history.append("events, _ = "
+            self.current["event_mappings"] = mappings
+            self.history.append("events, event_mappings = "
                                 "mne.events_from_annotations(data)")
```

Comment:
How does the history reflect the inversion of the mappings dict? It should somehow, otherwise copy and pasting the history won't work as a script.

Reply:
You're right. I missed it. Thanks!